src/topp/FileConverter.cpp

```diff
@@ -880,6 +886,15 @@ class TOPPFileConverter :
       SqMassFile sqm;
       sqm.store(out, exp);
     }
+    else if (out_type == FileTypes::OMS)
+    {
+      if (in_type != FileTypes::FEATUREXML)
+      {
+        OPENMS_LOG_ERROR << "Incompatible input data: FileConverter can only convert featureXML files to oms format.";
+        return INCOMPATIBLE_INPUT_DATA;
+      }
+      OMSFile().store(out, fm);
```

Comment:
Is the 
`IdentificationDataConverter::importFeatureIDs(fm)`
needed before storing the as OMS?